docs/common/vars.rst

```diff
@@ -10,11 +10,11 @@
 .. |whereabouts-version| replace:: v0.7.0
 .. |nvidia-ipam-version| replace:: v0.1.2
 .. |nic-feature-discovery-version| replace:: v0.0.1
-.. |sriovnetop-image-tag| replace:: network-operator-24.4.0
+.. |sriovnetop-image-tag| replace:: network-operator-24.4.1
 .. |sriovnetop-sriov-cni-image-tag| replace:: 3e6368077716f6b8368b0e036a1290d1c64cf1fb
 .. |sriovnetop-ib-sriov-cni-image-tag| replace:: fc002af57a81855542759d0f77d16dacd7e1aa38
 .. |sriovnetop-sriov-device-plugin-image-tag| replace:: e6ead1e8f76a407783430ee2666b403db2d76f64
 .. |node-feature-discovery-version| replace:: v0.13.2
-.. |helm-chart-version| replace:: v24.4.0
-.. |network-operator-version| replace:: 24.4.0
+.. |helm-chart-version| replace:: v24.4.1
```

Comment:
Do we have a new helm also?

Reply:
yes, we re-tagged images and updated helm chart too to have the same version for OCP and vanilla k8s